docs/content/integration/proxies/nginx-proxy-manager/index.md

```diff
@@ -217,6 +217,7 @@ location / {
     proxy_pass $forward_scheme://$server:$port;
 }
 ```
+{{< callout context="note" title="Websockets" icon="outline/info-circle" >}}Note that because we are using the advanced configuration tab, the switches on the `Details` tab will have no effect.  If websockets are required for a protected application, you must include the websocket.conf from the [NGINX Snippets](../nginx.md#supporting-configuration-snippets). {{< /callout >}}
```

Comment:
```suggestion
{{< callout context="note" title="Websockets" icon="outline/info-circle" >}}
Note that because we are using the advanced configuration tab, the switches on the `Details` tab will have no effect.  If websockets are required for a protected application, you must include the websocket.conf from the [NGINX Snippets](../nginx.md#supporting-configuration-snippets). 
{{< /callout >}}
```